lib/counter_culture/counter.rb

```diff
@@ -344,5 +358,60 @@ def attribute_was(obj, attr)
         end
       obj.public_send("#{attr}#{changes_method}")
     end
+
+    def assemble_money_counter_update(klass, id_to_change, quoted_column, operator, delta_magnitude)
+      update = "#{quoted_column} = COALESCE(CAST(#{quoted_column} as NUMERIC), 0)"
+
+      if Thread.current[:aggregate_counter_updates]
+        remember_counter_update(
+          klass,
+          id_to_change,
+          "#{update} +",
+          operator == '+' ? delta_magnitude : -delta_magnitude
+        )
+      else
+        "#{update} #{operator} #{delta_magnitude}"
+      end
+    end
+
+    def assemble_counter_update(klass, id_to_change, quoted_column, operator, delta_magnitude)
+      update = "#{quoted_column} = COALESCE(#{quoted_column}, 0)"
```

Comment:
I think `assemble_money_counter_update` and `assemble_counter_update` are the same except for the first line? Can we pull the common code out into a third method that both of these call?

Reply:
Done